src/components/Step.jsx

```diff
@@ -4,7 +4,8 @@ const Step = ({step, active, currentIndex}) => {
 
     return (
         <>
-            <div className={"step-number"}>{active ? "I'm active!" : currentIndex + 1}</div>
+            <div className={active ? "step-number active-background" : "step-number"}>{currentIndex + 1}</div>
```

Comment:
Is there a better way to write this?

Reply:
bring in the classnames library

https://www.npmjs.com/package/classnames